scripts/2Dhistograms.py

```diff
@@ -399,6 +406,7 @@ def load_data(
         event_counts.append((key, len(df)))  # Store the number of events and the key
 
     # Define ranking factors for each subfolder
+    ###FIXME: sumw has to change, whenever you run a different set. Pay attention!!!

```

Comment:
can you please comment it the former is xs and later is sumw ?

Reply:
also elaborate what is the ranking factor, I think it's to derived the yields, would be good to clarify